specs/linux/containerd_container_events.table

```diff
@@ -0,0 +1,19 @@
+table_name("containerd_container_events")
+description("Track containerd container events.")
+schema([
+    Column("container_id", TEXT, "Container ID"),
+    Column("time", BIGINT, "When the event has been fired by the event framework"),
+    Column("timestamp", TEXT, "Time of when the event happened"),
```

Comment:
What's the difference between time and timestamp?

Reply:
`timestamp` is the time reported by containerd, `time` is the time reported by osquery when the event was inserted into the ring buffer. It should be possible to remove timestamp and instead override the time in the event subscriber

Reply:
It seems like the osquery `time` is only in seconds which is substantially less precise than the containerd `timestamp` which is in nanoseconds

Reply:
After spending a bunch of time on this I just realized that the function that allows you to override the event time is private and intended only for tests. I'd prefer to only have the more precise timestamp but it could violate some assumptions about rows on other parts of the codebase and would require changes to `eventsubscriberplugin.cpp`